app/scripts/lib/mailcheck.js

```diff
@@ -55,37 +55,34 @@ define(function (require, exports, module) {
 
           // user got a suggestion to check their email input
           view.notifier.trigger('mailcheck.suggested');
```

Comment:
same as https://github.com/mozilla/fxa-content-server/pull/4703/files#r99847017 , except this should `logEvent` that mailcheck was suggested